avocado/utils/disk.py

```diff
@@ -40,6 +40,65 @@ def get_disk_blocksize(path):
     return fs_stats.f_bsize
 
 
+def create_loop_device(size, blocksize=4096, directory=''):
```

Comment:
if the default directory is the current directory, maybe this would be better: `directory='./'`